### ISSUE TITLE
[immagini] rotazione automatica

### ISSUE THREAD

**Issue by AlfioEmanueleFresta** [CroceRossaItaliana/jorvik#508]
Cito me stesso,:

> In breve, e' dovuto al fatto che la foto e' fatta con un dispositivo munito di accelerometro (es. cellulare, o piccola fotocamera) che individua il modo in cui e' stata scattata la foto. Il file JPEG e' di fatti sempre in orizzontale, ma nei metadati del file (EXIF) e' scritto che la foto e' stata scattata in verticale. 
>
> Alcuni programmi, come il visualizzatore di immagini di Windows, quando incontrano tale informazione nei metadati, mostrano la foto applicando la rotazione. Questo da l'impressione che il file sia correttamente ruotato, ma si tratta solo di un accorgimento fatto dal programma in fase di visualizzazione.
>
> La gente carica le fototessere e gli avatar in questo modo, convinte che siano ruotate correttamente.
>
> La foto e' quindi caricata in realta' orizzontalmente. Il programma che si occupa di generare le versioni rimpicciolite quadrate delle foto applica la rotazione prima di salvare l'anteprima sul server. Invece, il browser dell'utente, caricando la foto originale, non si occupa di mostrare la fotografia correttamente.
>
> Spesso le foto vengono rifiutate in processo di moderazione perche' al momento in cui il moderatore va a controllare il file originale della foto, questo si apre nel browser in formato originale, e viene mostrato orizzontale.
>

Quel che bisogna fare e' assicurarsi che, in seguito al caricamento, la fotografia venga salvata senza i dati EXIF ma ruotata alla vecchia maniera... in pixel. 

Data la libreria di gestione delle immagini che usiamo, dovrebbe essere tanto semplice quanto aggiungere uno step di processing che non faccia altro che ridimensionare e salvare la fotografia a rapporto 1.0 (apri, ruota, salva).

**Comment by yakky** [comment]
@AlfioEmanueleFresta se capisco bene: la fototessera originale andrebbe ruotata al caricamento in modo tale che l'orientamento rispetti le informazioni exif, e che l'originale sia comunque sostituito con la versione ruotata.
E' corretto?

**Comment by AlfioEmanueleFresta** [comment]
Esatto. Non dobbiamo memorizzare l'originale (orizzontale + EXIF) ma direttamente (verticale).
